sbt-assembly-all.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+#COMET_SPARK_VERSION=2.1.0 sbt ++2.11.10 assembly
+COMET_SPARK_VERSION=2.4.6 sbt ++2.11.10 assembly
```

Comment:
```suggestion
COMET_SPARK_VERSION=2.4.6 sbt ++2.11.12 assembly
```

Scala 2.11.10 is really ancient and there is a CVE open against it. Scala 2.11.12 should be the one flintstone we need.